dev-libs/angelscript/angelscript-2.35.1.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="A flexible cross-platform scripting library"
+DESCRIPTION="The AngelCode Scripting Library, or AngelScript as it is also known, \
+is an extremely flexible cross-platform scripting library designed to allow applications to \
+extend their functionality through external scripts. It has been designed from the beginning \
+to be an easy to use component, both for the application programmer and the script writer"
+HOMEPAGE="http://www.angelcode.com/angelscript/"
+COPYRIGHT="2003-2021 Andreas Jönsson"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="http://www.angelcode.com/angelscript/sdk/files/angelscript_2.35.1.zip"
+CHECKSUM_SHA256="5c1096b6d6cf50c7e77ae93c736d35b69b07b1e5047161c7816bca25b413a18b"
+SOURCE_DIR="sdk"
+PATCHES="angelscript-2.35.1.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	angelscript$secondaryArchSuffix
+	lib:libangelscript$secondaryArchSuffix
```

Comment:
There doesn't seem to be a compat version so I guess it's safe to use portVersion here:
```diff
- 	lib:libangelscript$secondaryArchSuffix
+ 	lib:libangelscript$secondaryArchSuffix = $portVersion
```